Dockerfile.centos

```diff
@@ -13,6 +13,7 @@ RUN go mod vendor
 ADD /base       /go/src/app/base
 ADD /manager    /go/src/app/manager
 ADD /listener   /go/src/app/listener
+ADD /vmaas_sync /go/src/app/vmaas_sync
```

Comment:
Need to be added also to Dockerfile.test. Please, try always `docker-compose -f docker-compose.test.yml up --build test` before requesting review.

Reply:
Fixed.